samples/matter/lock/src/lock.zap

```diff
@@ -18,13 +18,13 @@
   "package": [
     {
       "pathRelativity": "relativeToZap",
-      "path": "../../../../../modules/lib/matter/src/app/zap-templates/zcl/zcl.json",
+      "path": "../../../../../../matter/connectedhomeip/src/app/zap-templates/zcl/zcl.json",
```

Comment:
I think this is unexpected. We should keep the previous path that points to the NCS-supplied connectedhomeip repo.